src/Marian/index.ts

```diff
@@ -333,13 +332,19 @@ export default class Marian {
       return;
     }
 
-    const response = {
-      manifests: this.index.manifests.map((manifest) =>
-        new URL(`${this.index.manifestUrlPrefix}/${manifest.searchProperty}.json`).toString()
-      ),
-    };
+    let manifestList = '';
+    const openTags = '<a href=';
```

Comment:
minor nitpick. can we wrap these `<a>` tags in either `<div>`s or `<p>` so that they wrap in the page